xml/common_intro_feedback_i.xml

```diff
@@ -16,66 +16,69 @@
+  Your feedback and contribution to this documentation is welcome!
+  Several channels are available:
  </para>
 
  <variablelist>
   <varlistentry os="sles;sled">
-   <term>Bugs and Enhancement Requests</term>
+   <term>Service Requests and Support</term>
    <listitem>
-    <para os="sles;sled">
+    <para>
      For services and support options available for your product, refer to
      <link xlink:href="http://www.suse.com/support/"/>.
     </para>
     <para>
-     Help for &opensuse; is provided by the community. Refer to <link
-     xlink:href="https://en.opensuse.org/Portal:Support"/> for more information.
-    </para>
-    <para os="sles;sled">
-<!--taroth 2015-06-01: http://bugzilla.suse.com/show_bug.cgi?id=882059-->
-     To report bugs for a product component, go to
-     <link xlink:href="https://scc.suse.com/support/requests"/>, log in, and
-     click <guimenu>Create New</guimenu>.
-    </para>
-    <para os="osuse">
-     To report bugs &productname;, go to <link
-     xlink:href="https://bugzilla.opensuse.org/"/>, log in, and click
-     <guimenu>New</guimenu>.
+     To open a service request, you need a subscription at &scc;. Go to
+     <link xlink:href="https://scc.suse.com/support/requests"/>, log in, and click
+     <guimenu>Create New</guimenu>.
     </para>
    </listitem>
   </varlistentry>
-  <varlistentry os="osuse">
+  <varlistentry>
    <term>Bug Reports</term>
    <listitem>
     <para>
-     To report bugs for &productname;, go to
-     <link xlink:href="https://bugzilla.opensuse.org/"/>, log in, and
-     click <guimenu>New</guimenu>.
+      To report documentation bugs in <phrase os="sles;sled">
+      <link xlink:href="https://bugzilla.suse.com/"/>
+     </phrase><phrase os="osuse">
+      <link xlink:href="https://bugzilla.opensuse.org/"/>
+     </phrase>, use the links next to each headline in the HTML version of this
+     document. <guimenu>Report Documentation Bug</guimenu> opens Bugzilla,
+     preselects the right product version and category, and adds a link to the
+     specific section. You can start typing your bug report right away.
+     (A Bugzilla account is required).
     </para>
    </listitem>
   </varlistentry>
-  <varlistentry os="sles;sled">
-   <term>User Comments</term>
+  <varlistentry>
+   <term>Mail</term>
    <listitem>
     <para>
-     We want to hear your comments about and suggestions for this manual and
-     the other documentation included with this product. Use the User Comments
-     feature at the bottom of each page in the online documentation or go to
-     <link xlink:href="http://www.suse.com/documentation/feedback.html"/> and
-     enter your comments there.
+     Alternatively, you can report errors and send feedback concerning the
+     documentation to <literal>doc-team@suse.com</literal>. Make sure to include the
+     document title, the product version and the publication date of the
+     documentation. Refer to the respective section number and title (or
```

Comment:
I suggest "relevant" rather than "respective", as it's not really the correct way to use "respective".

Reply:
Always good to have a native speaker around! :) Fixed.